vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -94,22 +94,24 @@ The following is a list of specifications for the bundle descriptor and its comp
 ```
 
 ### Microservices Specifications
-|Name|Type|Required|Possible Values|Description|
+|Name|Type|Required|Possible Values [default]|Description|
 |:-|:-|:-|:-|:------------------------|
 |`name`|String|Yes||Microservice name|
 |`stack`|Enum|Yes|*spring-boot<br/>*node<br/>*custom|Microservice stack |
 |`dbms`|Enum|No|*none^  *embedded  *postgresql  *mysql  |DBMS required by the MS to provide services|
-|`ingressPath`|String|No||Custom ingress path|
 |`healthCheckPath`|String|No||Endpoint for a health check|
+|`healthCheckIngress`|Enum| Yes |*[canonical]  *custom| Use the canonical address or customize with `ingressPath`
+|`ingressPath`^^|String|No||Custom ingress path for health check|
 |`deploymentBaseName`|String|No||Used to define custom pod names|
 |`permissions`|[Permission[]](#permission-specification)|No| | List of permissions to grant to the microservice |
 |`roles`|String[]|No||Exposed security roles|
 |`env`|[EnvironmentVariable[]](#environmentvariables-specification)|No||Required environment variables|
 | |[Microservices Environment Variables](#microservices-environment-variables)|No||Entando-provided env variables for MS |
 |`commands`|[Command[]](#command-specification)|No||Custom command(s) definitions|
 |`version`|String|Required only for a custom stack||Microservice version override|
-> ^Note: Oracle and other DBMS types are not supported for automatic deployment in a container. Bundle env variables should be used instead, similar to connecting the EntandoApp to an [external database](../../tutorials/devops/external-db.md).
+> ^ dbms none: Oracle and other DBMS types are not supported for automatic deployment in a container. Bundle env variables should be used instead, similar to connecting the EntandoApp to an [external database](../../tutorials/devops/external-db.md).
 
+>^^ ingressPath: Customizing the ingress path for the health check requires an additional rewrite annotation or an API claim for the bundle. 
```

Comment:
@nshaw, this seems unsatisfactory. It didn't sound better to say the path works only partially for v5 bundles. If it's going to change soon, then I won't worry but if not, can I get more information about the workaround so we can add it to another section or something?

Reply:
@jyunmitch I would drop this line. We have a bug ticket for this but explaining the overall situation is a bit complicated. What we have works if you adjust the ingressPath - but it doesn't let you control the full path and that's the new feature/bug.